openfe/utils/visualization.py

```diff
@@ -116,24 +116,29 @@ def _draw_molecules(
     highlight_color: Tuple[float, float, float, float]
         RGBA tuple for the default highlight color used in the mapping
         visualization
+    atom_mapping: Dict[Tuple[int,int], Dict[int, int]], optional
+        used to align the molecules to each othter for clearer visualization, default None
     """
     if d2d is None:
         # select default layout based on number of molecules
         grid_x, grid_y = {
             1: (1, 1),
             2: (2, 1),
         }[len(mols)]
-        d2d = Chem.Draw.rdMolDraw2D.MolDraw2DCairo(grid_x*300, grid_y*300,
+        d2d = Chem.Draw.rdMolDraw2D.MolDraw2DCairo(grid_x * 300, grid_y * 300,
                                                    300, 300)
 
     # squash to 2D
     copies = [copy.deepcopy(mol) for mol in mols]
-    AllChem.Compute2DCoords(copies[0])
-    for (i, j), atomMap in atom_mapping.items():
-        try:
-            AllChem.GenerateDepictionMatching2DStructure(copies[j], copies[i])
-        except:
-            rms = AllChem.AlignMol(copies[j], copies[i], atomMap=[(k,v) for v,k in atomMap.items()])
+    [AllChem.Compute2DCoords(mol) for mol in copies]
+
+    if (atom_mapping is not None):
+        for (i, j), atomMap in atom_mapping.items():
+            try:
+                AllChem.GenerateDepictionMatching2DStructure(copies[j], copies[i])
+            except:
```

Comment:
Sorry my point here was badly worded, my main thing is "what error actually gets thrown here?" we should be covering that error specifically if we can rather than using base exceptions.

Reply:
Here the exception gets triggered:
![image](https://user-images.githubusercontent.com/12428005/176383517-83855739-3e6f-4077-87e4-ac0202229ff1.png)

Reply:
hm... removed the call in the try part (and the try except part as well)

Reply:
should resolve this and the next comment